.circleci/publish.sh

```diff
@@ -39,71 +39,71 @@ fi
 ########################################################################################
 ## Phase 3 - Publishing changed packages to NPM
 ########################################################################################
-printHeader "Phase 3 - Publishing changed packages to NPM"
-
-PUBLISHED_NPM_PACKAGES_COUNT=0
-
-for PACKAGE in ${CHANGED_PACKAGES}; do
-    echo -e "\n------------------------------------------------------------"
-    echo -e "Processing package: ${PACKAGE}..."
-
-    PACKAGE_PATH=${ROOT}/$PACKAGE
-    PACKAGE_NAME=$(cat ${PACKAGE_PATH}/package.json | jq -r ".name")
-    PACKAGE_CUR_VERSION=$(cat ${PACKAGE_PATH}/package.json | jq -r ".version")
-    PACKAGE_IS_PRIVATE=$(cat $PACKAGE_PATH/package.json | jq -r ".private // false")
-
-    echo -e "\tName: \t\t${PACKAGE_NAME}"
-    echo -e "\tPrivate: \t${PACKAGE_IS_PRIVATE}"
-    echo -e "\tCur version: \t${PACKAGE_CUR_VERSION}"
-
-    if [[ "$PACKAGE_IS_PRIVATE" = "true" ]]; then
-        echo -e "\n\tIGNORING private package"
-        continue
-    fi
-
-    ## increase patch version only
-    npm -w ${PACKAGE_NAME} version patch --no-git-tag-version --no-workspaces-update >/dev/null 2>&1
-
-    PACKAGE_NEW_VERSION=$(cat ${PACKAGE_PATH}/package.json | jq -r ".version")
-
-    echo -e "\tNew version: \t${PACKAGE_NEW_VERSION}"
-
-    echo -e "\n\tBuilding..."
-    npm -w ${PACKAGE_NAME} run build
-    echo -e "\n\tBuilding completed"
-
-    if [[ -n "$DRYRUN" ]]; then
-        echo -e "\nDryrun env var found - not actually publishing to NPM"
-        continue
-    fi
-
-    echo -e "Publishing..."
-
-    echo -e "---------------- PUBLISH START ----------------------\n"
-    # actual publish command
-    npm -w ${PACKAGE_NAME} publish --tag=latest --access public
-    PUB_SUCCESS=$?
-    echo -e "\n----------------- PUBLISH END -----------------------"
-
-    if [[ $PUB_SUCCESS -eq 0 ]]; then
-        PUBLISHED_NPM_PACKAGES_COUNT=$((PUBLISHED_NPM_PACKAGES_COUNT + 1))
-        TAG_NAME=${PACKAGE}_v${PACKAGE_NEW_VERSION}
-        echo -e "Successfully published package."
-        echo -e "Git staging '${PACKAGE_PATH}/package.json, committing and tagging with: '${TAG_NAME}'"
-        git add ${PACKAGE_PATH}/package.json
-        git commit -nm "[ci skip] CI/CD auto commit for '${PACKAGE}' NPM publish - parent commit: '${CIRCLE_SHA1}'"
-        git tag ${TAG_NAME}
-    else
-        echo -e "Error publishing package: ${PACKAGE} - exiting"
-        exit 1
-    fi
-done
-
-if [[ -n "$DRYRUN" ]]; then
-    echo -e "\nDryrun env var found - stopping script execution before 'Pushing commits to git'"
-    exit 0
-fi
-
+# printHeader "Phase 3 - Publishing changed packages to NPM"
```

Comment:
do we need commented code here, we always have git history, no? Any issue in removing it?

Reply:
readded Phase 3